kratos/python/add_vector_to_python.cpp

```diff
@@ -250,6 +260,14 @@ namespace Python
         CreateArray1DInterface< 6 >(m,"Array6");
         CreateArray1DInterface< 9 >(m,"Array9");
 
+        py::class_<Quaternion<double>, Quaternion<double>::Pointer>(m, "Quaternion")
+        .def(py::init<>())
+        .def_property("X", QuaternionGetX, QuaternionSetX)
+        .def_property("Y", QuaternionGetY, QuaternionSetY)
+        .def_property("Z", QuaternionGetZ, QuaternionSetZ)
+        .def_property("W", QuaternionGetW, QuaternionSetW)
```

Comment:
maybe add the `__str__` method like we have for many other classes

Reply:
done!